### ISSUE TITLE
[Request] Make a human readable modlist output

### ISSUE THREAD

**Issue by DrZed** [TanninOne/modorganizer#445]
I know anyone can read the actual output (mods > csv) but I want just a list that is readable cleanly, maybe just the mod names, or mod order and names?.

0  Unofficial Skyrim Patch
1  Unofficial Dawnguard Patch
2  Unofficial HearthFires Patch
3  Unofficial Dragonborn Patch
4  Unofficial HD Patch
....

**Comment by TanninOne** [comment]
Not sure if this isn't overkill. You could just open the csv in excel or openoffice / libreoffice to get a nicely readable view of the file.

**Comment by DrZed** [comment]
Yeah but I don't have any of those, and microsoft office stuff is garbageware to me, I use NP++

**Comment by simoniz0r** [comment]
Google docs opens CSV files just fine.

https://docs.google.com/spreadsheets/

**Comment by DrZed** [comment]
My point is, if I want to send a list of mods of my skyrim to a friend, it's easier if it's already human readable
instead of going to a stupid site by our lord and savior
or downloading a microshit software...

especially when my friend who I often send my working mod lists to is technologically inept and fails at even installing mod organizer without my help on teamviewer....

**Comment by simoniz0r** [comment]
Actually, Google Docs makes sending the list to your friend easier.  When you view it, it has already been uploaded, so all you have to do is give your friend the link to the file in your Google Docs.

For example, here's my mod list which got uploaded earlier when I tested to make sure that this worked before actually posting about it:
https://docs.google.com/spreadsheets/d/1z70R1IRUAFUDw6oQlkMjUKLmb1oVS-9OfKDxRJJi4Ho/edit?usp=sharing

**Comment by DrZed** [comment]
Fine I'll start using it, probably?
